jwst/resample/resample_utils.py

```diff
@@ -124,17 +125,11 @@ def shape_from_bounding_box(bounding_box):
 def calc_gwcs_pixmap(in_wcs, out_wcs, shape=None):
     """ Return a pixel grid map from input frame to output frame.
     """
-    if shape:
-        bb = wcs_bbox_from_shape(shape)
-        log.debug("Bounding box from data shape: {}".format(bb))
```

Comment:
would this debug message still be helpful to keep around?

Reply:
I don't see what for.